### PR TITLE
Add rubocop checking

### PR DESCRIPTION
**Pull request by tmtmtmtm**
Set up our default Rubocop checking, and make Travis test that everything is clean.

We had to turn off one check: we alias the `Everypolitician` class as `EveryPolitician` too. 
That's a feature, not a bug, even though it doesn’t match standard naming practice. 

(But that’s why we alias it, rather than just calling the class EveryPolitician by default)

### CODE REVIEW

File: test/everypolitician_test.rb [hunk 1]
```diff
@@ -190,16 +191,15 @@ def test_partial_dates
...
+      assert_equal 18_031, uganda_parliament.statement_count
     end
   end
 
   def test_lastmod_is_a_time
-    legislature = Everypolitician::Legislature.new({lastmod: '1469382925'}, nil)
+    legislature = Everypolitician::Legislature.new({ lastmod: '1469382925' }, nil)
     assert_equal 2016, legislature.lastmod.year
-    assert_equal    7, legislature.lastmod.month
+    assert_equal 7, legislature.lastmod.month
```
Comment (tmtmtmtm) [inline test/everypolitician_test.rb:201]: I'm not sure why it 'corrected' this to be less well matched up than before. It's now well matched to the line above, but not with the lines below.

**Comment by tmtmtmtm** [comment]
@octopusinvitro I ended up rewriting one of your helper functions, so it would be good for you to check that everything looks OK. `rubocop` just wanted me to rename it, but I figured I might as well take the opportunity to promote it to a full method, rather than just a utility function. (This also makes `reek` happier too.)

File: test/everypolitician_test.rb [hunk 2]
```diff
@@ -190,16 +191,15 @@ def test_partial_dates
   def test_expose_the_statement_count_of_a_legislature
     VCR.use_cassette('countries_json') do
       uganda_parliament = Everypolitician.country(slug: 'Uganda').legislature(slug: 'Parliament')
-      assert_equal 18031, uganda_parliament.statement_count
+      assert_equal 18_031, uganda_parliament.statement_count
```
Comment (octopusinvitro) [inline test/everypolitician_test.rb:194]: Wow

File: lib/everypolitician.rb [hunk 3]
```diff
@@ -108,25 +108,22 @@ def popolo
...
       @index_by_sources[dir][0]
     end
 
-    private
-
-    def to_UTC_time(string_time)
-      Time.at(string_time.to_i).gmtime
+    def lastmod
+      Time.at(lastmod_str.to_i).gmtime
     end
```
Comment (octopusinvitro) [inline lib/everypolitician.rb:126]: I really like the philosophy behind this change! :+1: 
However not sure about calling the field `lastmod_str`, as it reminds me of the [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) :laughing: 
Naming is hard!

**Comment by octopusinvitro** [comment]
Looks good to me. :+1: 
I am curious about what Rubocop wanted to rename that tiny method to :smile:

**Comment by tmtmtmtm** [comment]
rubocop doesn't make suggestions for _what_ to rename to, unfortunately :)

It just didn't like the lowercase_UPPERCASE_lowercase nature of the prior one!

File: lib/everypolitician.rb (hunk 3 above)
Comment (tmtmtmtm) [inline lib/everypolitician.rb:126]: Yeah, I wasn't too thrilled by it either, but in the absence of a better suggestion, I think it sufficiently conveys what we need. Definitely open to improving it further in later changes though!

**Pull request merged**